corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views/query.js

```diff
@@ -705,7 +706,7 @@ hqDefine("cloudcare/js/formplayer/menus/views/query", function () {
                 childView.clear();
             });
             self.setStickyQueryInputs();
-            if (self.dynamicSearchEnabled) {
+            if (self.dynamicSearchEnabled || this.searchOnClear) {
```

Comment:
nit: if `searchOnClear` is only true when `dynamicSearchEnabled` is enabled, then this can just be `dynamicSearchEnabled`

Reply:
gotcha-- cool makes sense. Ignore this comment then